sales.html

```diff
@@ -8,6 +8,28 @@
     <link href="css/style.css" type="text/css" rel="stylesheet" />
   </head>
   <body>
+
+    <form id="storeForm">
+      <fieldset>
+        <legend>Add a New Store!</legend>
+
+        <label for="name">Name of Location: </label>
+        <input name="name" type="text" size="15" />
+
+        <label for="minCustomers">Min Customers Per Hour: </label>
+        <input name="minCustomers" type="text" />
```

Comment:
Just so you know you can use `type="number"` to ensure that the user enters a number value.